packages/contracts-bedrock/deploy/013-OptimismPortalImpl.ts

```diff
@@ -13,12 +13,28 @@ const deployFn: DeployFunction = async (hre) => {
     'L2OutputOracleProxy'
   )
 
+  const finalSystemOwner = hre.deployConfig.finalSystemOwner
+  const finalSystemOwnerCode = await hre.ethers.provider.getCode(
+    finalSystemOwner
+  )
+  if (finalSystemOwnerCode === '0x') {
+    console.log(
+      `WARNING: setting OptimismPortal.GUARDIAN to ${finalSystemOwner} and it has no code`
```

Comment:
Feels like this should hault if done on a live deployment, no?

Reply:
Lemme check out the `isLiveDeployer` logic used in the other deployment files